__tests__/app.test.js

```diff
@@ -164,4 +164,50 @@ describe('app()', () => {
       });
     });
   });
+  describe('POST', () => {
+    describe('/api/articles/:article_id/comments', () => {
+      it('201: should respond with status 201 and a body object containing the inserted comment inside the correct article (1)', () => {
+        const newComment = {
+          username: 'icellusedkars',
+          body: 'Really great work on this ticket!',
+        };
+        return request(app)
+          .post('/api/articles/1/comments')
+          .send(newComment)
+          .expect(201)
+          .then(({ body }) => {
+            const { comment } = body;
+            expect(comment).toHaveProperty('author', expect.any(String));
+            expect(comment).toHaveProperty('body', expect.any(String));
+            expect(comment.body).toBe(newComment.body);
+          });
```

Comment:
Please add a test that shows that any unnecessary properties included in the comment are ignored. If for example, newComment = {username: 'icellusedkars', body: 'Really great work on this ticket!', banana: true}